recipes/default.rb

```diff
@@ -23,6 +23,7 @@
 %w(bencode i18n transmission-simple activesupport).each do |pkg|
   chef_gem pkg do
     action :install
+    compile_time true 
```

Comment:
This needs to be guarded so we don't break older versions of chef that lacked this attribute.

compile_time true if Chef::Resource::ChefGem.method_defined?(:compile_time)